Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -3498,6 +3503,25 @@ instance (priority := 100) Subgroup.normal_subgroupOf {H N : Subgroup G} [N.Norm
 #align subgroup.normal_subgroup_of Subgroup.normal_subgroupOf
 #align add_subgroup.normal_add_subgroup_of AddSubgroup.normal_addSubgroupOf
 
+theorem Subgroup.normalClosure_map_equiv_eq_normalClosure_comap {s : Set G} (f : G ≃* N) :
+    (Subgroup.normalClosure s).map f = Subgroup.normalClosure (f '' s) := by
```

Comment:
Can you also show the similar (and perhaps easier to prove)
```lean
    (Subgroup.normalClosure s).comap f = Subgroup.normalClosure (f ⁻¹' s) := by
```

Reply:
Good idea. I think `Subgroup.normalClosure (f ⁻¹' s) ≤ (Subgroup.normalClosure s).comap f` holds for any homomorphism `f : G →* N`.